scalableminds/webknossos
# upload dataset view sends organization now instead of team

Fixed the upload dataset view to work with organizations. Previously, all datasets were assigned to a team by default. Now datasets belong directly to an organization.

### Steps to test:
- Go to Dashboard --> Gallery View --> "Upload Dataset"
- Upload a dataset. --> should work
- Testing Datasets can be downloaded from the wiki

### Issues:
- fixes #2473

------
